Fix datadep for run_post

chunhuazhou opened the pull request:
## DESCRIPTION OF CHANGES: 
Change run_post data dependency to RESTART/coupler.res to avoid kicking off run_post before restart files are ready. 

## TESTS CONDUCTED: 

## DEPENDENCIES:

## DOCUMENTATION:

## ISSUE (optional): 

## CONTRIBUTORS (optional):

chunhuazhou committed:
Fix datadep for run_post

chunhuazhou commented:
This commit is to fix the problem when run_post run successfully but no restart files under nwges/

hu5970 reviewed (commented):
which task this dependents for?
The UPP cannot depend on restart because most of forecast hour has not restart files.

chunhuazhou committed:
Update FV3LAM_wflow.xml

Add and extend the age of logf* files for run_post datadep, in case need to rerun the run_post jobs after moving the restart file from step to nwges

chunhuazhou commented:
> which task this dependents for?
> The UPP cannot depend on restart because most of forecast hour has not restart files.

This is for run_post. It depends on fcst_fv3lam/logf* but occasionally logf* files are ready and run_post finishes without the restart files ready. In this situation, prep_cyc can not run due to lack of restart files under nwges/

chunhuazhou commented:
> > which task this dependents for?
> > The UPP cannot depend on restart because most of forecast hour has not restart files.
> 
> This is for run_post. It depends on fcst_fv3lam/logf* but occasionally logf* files are ready and run_post finishes without the restart files ready. In this situation, prep_cyc can not run due to lack of restart files under nwges/

I just added back dependency on logf* files but extended the file age.

hu5970 commented:
@chunhuazhou Can we just using logf* file but an extended file age. We need to figure out a better way to solve this problem.

chunhuazhou commented:
> @chunhuazhou Can we just using logf* file but an extended file age. We need to figure out a better way to solve this problem.

For the real-time ensemble run, for now I think the dependency on one of the three files works the best. If we only have logf*, this extended file age would delay the whole workflow and we don't know how long is the best. If we only have dependency on the restart files, rerunning run_post won't kick off if previous try has already moved the restart files to nwges.

hu5970 commented:
@chunhuazhou Let's use those dependence for HWT experiments. We need to figure out a better way to save restart files.
